vllm/attention/backends/abstract.py

```diff
@@ -1,11 +1,18 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass, fields
+from enum import Enum, auto
 from typing import (Any, Dict, Generic, List, Optional, Set, Tuple, Type,
                     TypeVar)
 
 import torch
 
 
+class AttentionType(Enum):
```

Comment:
Could you export **AttentionType** type in `vllm/attention/__init__.py` just as **AttentionMetadata**? It's convinient to import **AttentionType** from model file.

Reply:
Thanks, done.